docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,259 @@ Postprocessing
+    >>> data = fetch_openml(data_id=1590, as_frame=True)
+    >>> X = pd.get_dummies(data.data)
+    >>> y = (data.target == '>50K') * 1
+    >>> sex = data.data['sex']
+    >>> logistic_regression = LogisticRegression()
+    >>> threshold_optimizer = ThresholdOptimizer(
+    ...     estimator=logistic_regression,
+    ...     constraints="demographic_parity",
+    ...     objective="accuracy_score")
+    >>> threshold_optimizer.fit(X, y, sensitive_features=sex)
+    ThresholdOptimizer(estimator=LogisticRegression())
+    >>> threshold_optimizer.interpolated_thresholder_._print_thresholds()
+    {
+        "Female": {
+            "p0": 0.9527202201546324,
+            "operation0": "[>0.5]",
+            "p1": 0.047279779845367575,
+            "operation1": "[>-inf]"
+        },
+        "Male": {
+            "p0": 0.9998874603553525,
+            "operation0": "[>0.5]",
+            "p1": 0.00011253964464752464,
+            "operation1": "[>-inf]"
+        }
+    }
+    >>> plot_threshold_optimizer(threshold_optimizer)
+
+Notice that the thresholding rules do not include :math:`p_{\text{ignore}}` which is
+only relevant for :code:`equalized_odds`. The results can be interpreted as
+follows:
+
+- "Male":
+
+  - if the score is less or equal to :math:`0.5` predict 1 with probability
+    :math:`0.00011`
+  - if the score is above :math:`0.5` predict 1
+
+- "Female:
+
+  - if the score is less or equal to :math:`0.5` predict 1 with probability
+    :math:`0.04728`
+  - if the score is above :math:`0.5` predict 1
+
+The thresholds are actually the same for both groups here, which
+does not always have to be true. As a consequence, the only difference is
+the probability to predict label 1 in the case where the score is below
+:math:`0.5`.
+
+Note that the chart is different from the chart for :code:`equalized_odds`.
+It shows :code:`selection_rate` (which is the basis for our fairness
+criterion) on the x-axis :code:`demographic_parity`, and
+:code:`accuracy_score` (which is the basis for the implicit performance
+objective) on the y-axis.
+
+The following combinations of fairness criteria and objectives are available:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   *  - fairness criteria
+      - objectives
+   *  - :code:`equalized_odds`
+      - :code:`accuracy_score`, :code:`balanced_accuracy_score`
+   *  - :code:`demographic_parity`, :code:`false_positive_rate_parity`, :code:`false_negative_rate_parity`, :code:`true_positive_rate_parity`, :code:`true_negative_rate_parity`
+      - :code:`selection_rate`, :code:`true_positive_rate`, :code:`true_negative_rate`, :code:`accuracy_score`, :code:`balanced_accuracy_score`
+
+Using a pre-trained estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, :class:`ThresholdOptimizer` trains the passed estimator using its
+:code:`fit()` method. In some cases we already have a trained estimator, for
+example if we want to compare it with others. For this use case
```

Comment:
not sure what "for example if we want to compare it with others" means here.

Reply:
I rephrased that a little and removed this particular sentence. I think what I meant originally was that one might want to compare several mitigation techniques and the pretrained model, but that's not a great justification for this either.